### PR TITLE
Add p0 pyramid quadrature support.

### PR DESCRIPTION
**Pull request by tarikdzanic**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Tarik Dzanic <tdzanic@tamu.edu>
```
Comment (FreddieWitherden) [inline AUTHORS]: Move down a line to maintain alphabetical order. (PS You can use git amend and force-push to update your PR without having to open a new one.)

**Comment by FreddieWitherden** [comment]
Is it possible to make this rule -d1 by adding a weight?

**Comment by tarikdzanic** [comment]
Just for p0 or also for all GL pyramid quadratures? Since the rest of them don't have weights.

**Comment by FreddieWitherden** [comment]
Ah yes, it appears as if I never got around to calculating them.  Lets leave as is for the moment.

File: AUTHORS (hunk 1 above)
Comment (FreddieWitherden) [reply AUTHORS]: Are you okay to tweak this?

Comment (tarikdzanic) [reply AUTHORS]: I think I did it properly, can you confirm?

**Comment by FreddieWitherden** [comment]
Looks good.  You'll need to resubmit against develop before we can merge (master is only used for brining forward releases).

**Pull request merged**